File: Proyectos Python (2)/Temperatura promedio.py
print ("Programa que va a calcular la temperatura promedio en una semana")
t1 = int( input ("Ingrese la temperatura del primer dia: "))
t2 = int ( input ("Ingrese la temperatura del segundo dia: "))
t3 = int ( input ("Ingrese la temperatura del tercer dia: "))
t4 = int ( input ("Ingrese la temperatura del cuarto dia: "))
t5 = int ( input ("Ingrese la temperatura del quinto dia: "))
t6 = int ( input ("Ingrese la temperatura del sexto dia: "))
t7 = int ( input ("Ingrese la temperatura del septimo dia: "))
s = int (t1+t2+t3+t4+t5+t6+t7)

if s/7>35:
    print("Que semana tan calurosa, la temperatura promedio durante toda la semana fue de: ",(s/7))
else:
    print("")
    if s/7>=15:
        print("que clima tan delicioso, la temperatura promedio durante toda la semana fue de: ",(s/7))
    else:
        print ("Que semana tan fria, la temperatura promedio durante toda la semana fue de: ",(s/7))
        

File: Proyectos  c++ (2)/Temperatura promedio.cpp
#include<iostream>
#include<locale.h>

using namespace std;

int main(){
setlocale(LC_CTYPE, "spanish");
		int d1 , d2 , d3 , d4  , d5 , d6 , d7 , s , pr  ;
		cout<< "Vamos calcular la temperatura promedio en una semana" <<endl;
		cout<< "Ingrese la temperatura en el primer dia: " <<endl;
		cin>> d1;
		cout<< "Ingrese la temperatura en el segundo dia: " <<endl;
		cin>> d2;
		cout<< "Ingrese la temperatura en el tercer dia: " <<endl;
		cin>> d3;
		cout<< "Ingrese la temperatura en el cuarto dia: " <<endl;
		cin>> d4;
		cout<< "Ingrese la temperatura en el quinto dia: " <<endl;
		cin>> d5;
		cout<< "Ingrese la temperatura en el sexto dia: " <<endl;
		cin>> d6;
		cout<< "Ingrese la temperatura en el septimo dia: " <<endl;
		cin>> d7;
		s = d1 + d2 + d3 + d4 + d5 + d6 + d7;
		pr = s / 7;
		
			if (pr>35)
					{
						cout<<"Que semana tan calurosa, el promedio de temperatura fue: "<<pr ;
					}
	    			else 
	    			{
	    				cout<<"";
						if(pr>=15)
						{
							cout<<"Que clima tan deliciso, la temperatura fue: "<<pr; 
						}
						else
						{
							cout<<"Que semana tan fria, promedio de la temperatura fue: "<<pr;
						}
					}
		

		
		
}
